pkg/address/testing.go

```diff
@@ -0,0 +1,20 @@
+package address
+
+import (
+	"fmt"
+)
+
+// NewForTestGetter returns a closure that returns an address unique to that invocation.
+// The address is unique wrt the closure returned, not globally.
+func NewForTestGetter() func() Address {
```

Comment:
I suggest removing this too. It's not really library code, can easily be re-implemented (possibly differently) by others.